tests/functional/pv/pv_services/test_rwop_pvc.py

```diff
@@ -59,6 +59,13 @@ def test_pod_with_same_priority(self, pod_factory, interface):
 
         # Verify that PVCs are reusable by creating new pods
         log.info(f"PVC obj {self.pvc_obj}")
+        self.create_pod_and_validate_pending(pod_factory, interface)
+
+        self.pvc_obj.resize_pvc(20, True)
+
+        self.create_pod_and_validate_pending(pod_factory, interface)
+
+    def create_pod_and_validate_pending(self, pod_factory, interface):
```

Comment:
Nested function is not actually a  necessity here. It will be nice to keep it outside the test function and pass the function arguments.